examples/BodySegmentation-maskbackground/sketch.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020-2023 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ml5 Example
+BodyPix
+=== */
+
+let bodypix;
+let video;
+let segmentation;
+
+let options = {
+  outputStride: 16, //adjust the output stride and see which one works best!
+  multiSegmentation: false,
+  segmentBodyParts: true,
+  flipHorizontal: true,
```

Comment:
Hi Dan! If your question meant "why these four paramters", that's the latter three (multiSegmentation, segmentBodyParts, flipHorizontal) actually depends case by case. For example, if the user wants to detect multiple people and output different segmentation, then multiSegmentation needs to be turned on. As for outputStride, that's because I noticed this parameter could affect model performance, in terms of accuracy, so I thought this might be a good way to familiarize our users to machine learning. 
I have set a default values for all these in index.js. We can create a default input for different cases like webcam or image for parameters like flipHorizontal and segmentBodyParts, but I think it might be a good idea to leave multiSegmentation (one segmentation for multiple people or multiple segmentation) to the user?
Lemme know if there's still something I didn't explain clearly or understand wrongly! Be happy to elaborate/correct :)